docs/content/dagster-cloud/developing-testing/branch-deployments/using-branch-deployments-with-github.mdx

```diff
@@ -92,7 +143,40 @@ locations:
       registry: 764506304434.dkr.ecr.us-east-1.amazonaws.com/branch-deployment-agent
 ```
 
-### Step 4.2: Configure GitHub Action secrets
+### Step 2.3: Configure GitHub Action secrets
+
+In this step, you'll configure the secrets used by the Branch Deployments GitHub Action. The steps and required secrets vary a bit depending on the type of deployment you're using.
+
+Select your deployment type below to view instructions.
+
+<details>
```

Comment:
Based on my testing, it seems like this step is required if the user is adding another GH-based code location after they complete onboarding.

I assumed that even if the connected GH repo had a `dagster_cloud.yaml` file, they'd still have to do this part manually, but didn't have time to fully test this. Confirmation needed pls.